fv_be/app/models/sites.py

```diff
@@ -0,0 +1,150 @@
+from django.core.validators import validate_slug
+from django.db import models
+from django.utils.translation import gettext as _
+
+from fv_be.app.models import BaseModel
+from fv_be.users.models import User
+
+from .constants import Role, Visibility
+
+
+class LanguageFamily(BaseModel):
+    """
+    Represents a Language Family.
+    """
+
+    # from FVLanguageFamily type
+
+    class Meta:
+        verbose_name = _("language family")
+        verbose_name_plural = _("language families")
+
+    # from parent_languages.csv OR from fvdialect:language_family on one of the
+    # linked sites (via one of the linked languages)
+    title = models.CharField(max_length=200)
+
+    # from dc:title
+    alternate_names = models.TextField(max_length=200, blank=True)
+
+    def __str__(self):
+        return self.title
+
+
+class Language(BaseModel):
+    """
+    Represents a Language.
+    """
+
+    # from fvlanguage type
+
+    class Meta:
+        verbose_name = _("language")
+        verbose_name_plural = _("languages")
+
+    # from parent_languages.csv OR from fvdialect:parent_language on one of the linked sites
+    title = models.CharField(max_length=200)
+
+    # from dc:title
+    alternate_names = models.TextField(max_length=200, blank=True)
+
+    # from fva:family
+    language_family = models.ForeignKey(LanguageFamily, on_delete=models.PROTECT)
+
+    # from fvdialect:bcp_47
+    # BCP 47 Spec: https://www.ietf.org/rfc/bcp/bcp47.txt
+    language_code = models.TextField(max_length=100, blank=True)
```

Comment:
I think 20 would be a better (very generous) max. These are usually short, e.g. `en`, `haa`, `kwk-napa`. And maybe CharField rather than TextField.

Reply:
Thanks! Changed it.